Documentation/ceph-object-bucket-claim-crd.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim CRD
```

Comment:
We are really introducing a new concept to the admin. He has never heard of an object bucket claim before. We need to really explain what this feature is, what scenario it solves for the admin, and then explain a little about how it works. The current document tells me the details of CRD settings, but it doesn't give all the background and description that is really needed for this new feature.

Reply:
Done